examples/hosts/app-integration/schema-upgrade/src/migrator/migrator.ts

```diff
@@ -60,6 +73,12 @@ export class Migrator extends TypedEventEmitter<IMigratorEvents> implements IMig
      * that a freshly-loaded migrated container is in collaborating state.
      */
     private readonly takeAppropriateActionForCurrentMigratable = () => {
+        if (!this.connected) {
```

Comment:
We don't really have to be connected to `ensureLoading()` -- maybe this should be scoped to `ensureMigrating()`?